src/ReactLeafletMap/layers/TiledVectorLayer/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import {withLeaflet} from "react-leaflet";
+import memoize from "memoize-one";
+
+import Tile from "./Tile";
+
+/**
+ * @param uniqueLayerKey {string}
+ * @param tile {Object}
+ * @return {string}
+ */
+function getTileKey(uniqueLayerKey, tile) {
+	return `${uniqueLayerKey}_${tile.level}_${JSON.stringify(tile.tile)}`;
+}
+
+/**
+ * @param uniqueLayerKey {string}
+ * @param tiles {Array}
+ * @param fidColumnName {string}
+ * @return {[]} A collection of feature keys grouped by tile key
+ */
+function getFeatureKeysGroupedByTileKey(uniqueLayerKey, tiles, fidColumnName)  {
+	let groupedKeys = [];
+	_.forEach(tiles, tile => {
+		groupedKeys.push({
+			tileKey: getTileKey(uniqueLayerKey, tile),
+			featureKeys: tile.features.map(feature => feature.id || feature.properties[fidColumnName])
```

Comment:
In case of design is this better solution to pass only featureKeys. But in case of performance could be better to pass whole features and safe forEach on all features.

Just proposal, not confirmed by measurement